build/Nuget/Microsoft.NET.Sdk.Nuget.targets

```diff
@@ -19,142 +19,53 @@
       </CopyLocalAssembly>
 
       <CopyLocalAssembly Include="System.Runtime.Serialization.Primitives">
         <Version>4.1.1</Version>
         <TFM>netstandard1.3</TFM>
         <DestinationTFM>netcoreapp1.0</DestinationTFM>
       </CopyLocalAssembly>
-
-      <CopyLocalAssembly Include="System.Runtime.InteropServices.RuntimeInformation">
-        <Version>4.0.0</Version>
-        <TFM>net45</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Security.Cryptography.Algorithms">
-        <Version>4.2.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Security.Cryptography.Primitives">
-        <Version>4.0.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Threading.Thread">
-        <Version>4.0.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.IO.FileSystem">
-        <Version>4.0.1</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.AppContext">
-        <Version>4.1.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Console">
-        <Version>4.0.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Diagnostics.FileVersionInfo">
-        <Version>4.0.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Diagnostics.Process">
-        <Version>4.1.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Diagnostics.StackTrace">
-        <Version>4.0.1</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.IO.Compression">
-        <Version>4.1.0</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.IO.FileSystem.Primitives">
-        <Version>4.0.1</Version>
-        <TFM>net46</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
+      
       <CopyLocalAssembly Include="System.Security.Claims">
         <Version>4.0.1</Version>
         <TFM>net46</TFM>
         <DestinationTFM>net46</DestinationTFM>
       </CopyLocalAssembly>
-      <CopyLocalAssembly Include="System.Security.Cryptography.Encoding">
```

Comment:
I did a diff of the files that were previously included in the tools\net46 directory, and what was in the net46 output of the tasks build.  These were the ones which were missing.

I think that we probably don't need them, but at this point it seemed less risky to leave them in.